Signal/src/views/ContactTableViewCell.m

```diff
@@ -75,13 +75,9 @@ - (void)configureWithContact:(Contact *)contact contactsManager:(OWSContactsMana
         blockedLabel.textAlignment = NSTextAlignmentRight;
         blockedLabel.text
             = NSLocalizedString(@"CONTACT_BLOCKED_INDICATOR", @"An indicator that a contact has been blocked.");
-        blockedLabel.font = [UIFont ows_mediumFontWithSize:self.nameLabel.font.pointSize];
-        blockedLabel.textColor = [UIColor blackColor];
-        [self addSubview:blockedLabel];
+        blockedLabel.font = [UIFont ows_mediumFontWithSize:13.f];
+        blockedLabel.textColor = [UIColor colorWithWhite:0.5f alpha:1.f];
         [blockedLabel sizeToFit];
-        [blockedLabel autoVCenterInSuperview];
-        [blockedLabel autoPinEdgeToSuperviewMargin:ALEdgeRight];
-        [blockedLabel autoPinEdge:ALEdgeLeft toEdge:ALEdgeRight ofView:self.textLabel];
```

Comment:
> I like that you're using accessoryView, but there's an issue - we can't do that in the "new/edit group convo" view because we use "accessory" checkboxes there to indicate which users you're adding to the group.

Wait, I guess this isn't an issue.  A given signal user is either (already in the group/added to the group/blocked/none of the above).  A blocked user can't be in the group, so there's no conflict in terms of the accessories.